sphinx/api/built-in/rose_bunch.rst

```diff
@@ -167,3 +171,18 @@ sections of the :rose:file:`rose-app.conf` file, but
          ``command-instances`` and ``COMMAND_INSTANCES``, which are reserved
          for the auto-generated list of instances when the
          :rose:conf:`[bunch]command-instances=N` option is used.
+
+.. _roseBunch.CylcTasks:
+
+Rose Bunch in Cylc Tasks
+------------------------
+
+Rose Bunch examines environment variables set in the
+:ref:`Cylc job script variables<Task Job Script Variables>` to produce
+sensible behavior in Rose Bunch tasks if they are re-run by Cylc:
+
+- If a Cylc task is re-submitted as part of a new flow *all* jobs will be run.
+
+- If a Cylc task is re-submitted in the same flow as it previously ran
+  incremental mode will be switched on by default; so jobs which have
+  previously succeeded will be skipped.
```

Comment:
This is a little unclear to me, suggest something along the lines of:

```rest
Incremental Mode In Cylc Tasks
---

When incremental mode is turned on, only failed commands will be re-run if the task is run again (e.g. by manual triggering, or automatic retries).

If the task is run again as part of a new flow (e.g. `--flow=new`), then all commands will be re-run.
```

Reply:
Happy with suggestion, taken it.